src/common/components/repositories-list/index.js

```diff
@@ -15,15 +15,6 @@ import { spinner as spinnerStyles } from '../../containers/container.css';
 const SNAP_NAME_NOT_REGISTERED_ERROR_CODE = 'snap-name-not-registered';
 
 class RepositoriesList extends Component {
-
-  componentDidMount() {
-    const { authenticated } = this.props.auth;
-
-    if (authenticated) {
-      this.props.dispatch(fetchUserSnaps());
-    }
-  }
-
```

Comment:
`fetchUserSnaps` is also dispatched in component `RepositoriesHome`, so I removed the duplicate.